### PR TITLE
Cleanup of factory reference manual section

### PR DESCRIPTION
**Pull request by kprosise**
Style guide lines applied. Removed link to the Yocto Project layers section which is part of the LmP reference manual; this was done to make navigation less confusing, and to keep sections focused.

The section on Factory Sources was placed before the Factory Definition, for better explanatory flow.

QA steps: Ran linter and addressed issues where possible/appropriate. Checked html output in browser, no obvious rendering issues discovered. Ran linkcheck.

This commit addresses FFTK-2790
This commit addresses FFTK-1880
This commit applies to FFTK-2732
This commit applies to FFTK-2730
This commit applies to FFTK-988

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for bc84a46e13691a07394e85d5a86200bca8603010 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2484/docs/artifacts/html/index.html

**Comment by quaresmajose** [comment]
The image `lmp-base-console-image` from the public LmP is mentions in some places but I think the default image used in every  factory is the `lmp-factory-image`.

**Review by angolini — APPROVED** [review]
Only few suggestion

File: source/reference-manual/factory/factory-sources.rst [hunk 1]
```diff
@@ -15,114 +13,96 @@ Navigate to https://source.foundries.io/factories/<factory>/
...
 
 .. prompt:: bash host:~$
 
     cd meta-subscriber-overrides
-    git checkout devel
-    git checkout -b new_branch
-    git commit -m "[skip ci] create new branch" --allow-empty
-    git push --set-upstream origin new_branch
+    git checkout main
+    git checkout -b devel
```
[the remark this answers is not in the document]
Comment (kprosise) [reply source/reference-manual/factory/factory-sources.rst:84]: Well I did mention "if you not yet have a devel branch", though mostly to save from having to change it and think up another name 😅. 

I can easily replace it with \`\`feature1\`\` though!

File: source/reference-manual/factory/factory-definition.rst [hunk 2]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
+	      # "params" overriden by what is specified after "refs/heads/devel:".
+	      # In the "devel" build, IMAGE will now equal "lmp-mini" rather than
+	      # "lmp-base-console-image" as initially defined.
+
+               lmp:
                  params:
-                   IMAGE: lmp-mini
+                  IMAGE: lmp-base-console-image
+                machines:
+                  - imx8mmevk
```
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:147]:
```suggestion
                  - imx8mn-ddr4-evk
```

File: source/reference-manual/factory/factory-definition.rst [hunk 3]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
+                ref_options:
+                  refs/heads/devel:
+                    machines:
+                      - raspberrypi4-64
```
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:151]:
```suggestion
                      - imx8mn-ddr4-evk
```

File: source/reference-manual/factory/sboms.rst [hunk 4]
```diff
@@ -6,23 +6,27 @@ Software Bill of Materials
...
 SBOMs and Builds
 ----------------
 
 The FoundriesFactory CI generates SBOM artifacts whenever there a change happens in a Factory build.
-this happens for two kinds of builds:
+This happens for two kinds of builds:
 
- * Yocto Project — `Software Package Data Exchange`_ (SPDX) artifacts using built-in tooling.
- * Container — produce SDPX artifacts using Syft_.
+ * Yocto Project— `Software Package Data Exchange`_ (SPDX) artifacts using built-in tooling.
```
Comment (kprosise) [inline source/reference-manual/factory/sboms.rst:28]:
```suggestion
 * Yocto Project: `Software Package Data Exchange`_ (SPDX) artifacts using built-in tooling.
```

File: source/reference-manual/factory/sboms.rst [hunk 5]
```diff
@@ -6,23 +6,27 @@ Software Bill of Materials
...
+ * Container—produce SDPX artifacts using Syft_.
```
Comment (kprosise) [inline source/reference-manual/factory/sboms.rst:29]:
```suggestion
 * Container: produce SDPX artifacts using Syft_.
```

**Review by kprosise — COMMENTED** [review]
review suggestions

File: source/reference-manual/factory/api-access.rst [hunk 6]
```diff
@@ -18,70 +18,81 @@ These credentials allow users to access:
...
- * ``source:read-update`` - Useful for Git.
- * ``targets:read, devices:read, ci:read`` - read-only access
-   for fioctl or REST API
- * ``targets:read-update, devices:read-update, ci:read`` - read-update
-   access for fioctl.
- * ``containers:read`` - Useful for running docker commands on
-   factory containers.
+ * ``source:read-update``: For Git.
+ * ``targets:read, devices:read, ci:read``: Read-only access for Fioctl or REST API.
+ * ``targets:read-update, devices:read-update, ci:read``: read-update access for Fioctl.
```
Comment (vanmaegima) [inline source/reference-manual/factory/api-access.rst:37]:
```suggestion
 * ``targets:read-update, devices:read-update, ci:read``: Read-update access for Fioctl.
```

File: source/reference-manual/factory/ci-webhooks.rst [hunk 7]
```diff
@@ -78,91 +63,89 @@ sender.
...
-A quick example
----------------
+A Docker-compose based project has been written to help prototype.
+This example uses Ngrok_ so that it can be tested behind firewalls.
 
-A simple project docker-compose based project has been written to
-help users prototype. This example uses Ngrok_ so that it can be
-tested behind firewalls.
+.. note:: Ngrok changes URLs every time it restarts.
+   This requires the factory-config.yml file to change as well.
```
Comment (vanmaegima) [inline source/reference-manual/factory/ci-webhooks.rst:107]:
```suggestion
   This requires the ``factory-config.yml`` file to change as well.
```

File: source/reference-manual/factory/factory-definition.rst [hunk 8]
```diff
@@ -26,10 +25,10 @@ Configures the users who receive an email with the build notification.
 notify:
  email:
   users: ``<email_1>,<email_2>,<...>``
-      **Required:** Comma separated list of addresses to email after each CI build.
+      **Required:** A Comma separated list of email addresses.
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:28]: I still think the old content is relevant. This lists the emails to notify a build has passed/fail.

File: source/reference-manual/factory/factory-definition.rst [hunk 9]
```diff
@@ -67,13 +66,13 @@ Variables to be used with the metadata and artifacts.
              EXAMPLE_VARIABLE_1: hello_world
            machines:
              - imx8mmevk
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:68]: Current machine name:
```suggestion
             - imx8mm-lpddr4-evk
```

File: source/reference-manual/factory/factory-definition.rst [hunk 10]
```diff
@@ -67,13 +66,13 @@ Variables to be used with the metadata and artifacts.
...
-           image_type: lmp-factory-image
+           image_type: lmp-base-console-image
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:69]: Hmm not really. `lmp-factory-image` is still the image to go for factories. `lmp-base-console-image` is the one from LmP.

File: source/reference-manual/factory/factory-definition.rst [hunk 11]
```diff
@@ -67,13 +66,13 @@ Variables to be used with the metadata and artifacts.
...
            ref_options:
-            refs/heads/devel:
+            refs/heads/main:
               machines:
-                - raspberrypi3-64
+                - raspberrypi4-64
               params:
-                IMAGE: lmp-mini
+                IMAGE: lmp-mini-image
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:75]:
```suggestion
                IMAGE: lmp-factory-image
```

File: source/reference-manual/factory/factory-definition.rst [hunk 12]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
- .. note::
+      **Default**: Set during :ref:`gs-signup`.
 
-     The CI is configured to decline changes in the ``machines:`` parameter.
-     If needed, ask a support engineer to update the machine definition for your
-     FoundriesFactory.
+     .. note::
+        
+        The CI is configured to decline changes to the ``machines:`` parameter.
+        If needed, ask a support engineer to update the machine definition for your Factory.
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:120]:
```suggestion
        If needed, ask a support engineer to update the machine definition for your Factory.
```
We need a link to support.foundries.io here

File: source/reference-manual/factory/factory-definition.rst [hunk 13]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
-      **Optional:** Set the LmP image type to produce by recipe name. For
-      example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.
+      **Optional:** Set the LmP image type by recipe name.
+      For example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.
 
-      **Default:** ``lmp-factory-image`` |br| (from
-      **recipes-samples/images/lmp-factory-image.bb** in your
-      **meta-subscriber-overrides.git** repo)
+      **Default:** ``lmp-base-console-image`` |br| 
+      from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:127]:
```suggestion
      **Default:** ``lmp-base-console-image`` |br| 
      from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```
This is wrong, please revert to the original.
base console comes from LmP - fast first target, however factory-config should point to the factory image which is included in meta-subscriber-overrides (there's no base console image there).
If it's not clear why, please let me know in Slack.

File: source/reference-manual/factory/factory-definition.rst [hunk 14]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
+	      # In the below example, when the branch named "devel" is built by our
+	      # CI system, it will have its option values for "machine" and
+	      # "params" overriden by what is specified after "refs/heads/devel:".
+	      # In the "devel" build, IMAGE will now equal "lmp-mini" rather than
+	      # "lmp-base-console-image" as initially defined.
+
+               lmp:
                  params:
-                   IMAGE: lmp-mini
+                  IMAGE: lmp-base-console-image
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:145]:
```suggestion
                  IMAGE: lmp-base-console-image
```
Revert

File: source/reference-manual/factory/factory-definition.rst [hunk 15]
```diff
@@ -230,22 +223,21 @@ Defines the container's configuration, including some image configuration and ta
...
+      If it is not specified or its value is empty, then all Target's apps are preloaded.
 
       **Default:**  None
 
  platforms:|br| ``- arm`` |br| ``- arm64`` |br| ``- amd64``
       **Optional:** Specify a list of architectures to build containers for.
       Containers are only built for the specified list.
 
-      **Default:** ``arm,arm64,amd64``
+      **Default:** Variable, set during Signup.
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:240]:
```suggestion
      **Default:** Variable, set during Signup. 
```
I didn't know about this one, for me it still defaults to the 3 archs.

File: source/reference-manual/factory/factory-definition.rst (hunk 8 above)
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:28]:
```suggestion
      **Required:** A Comma separated list of email addresses to email after each CI build.
```

File: source/reference-manual/factory/factory-definition.rst (hunk 10 above)
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:69]:
```suggestion
           image_type: lmp-factory-image
```

File: source/reference-manual/factory/factory-definition.rst (hunk 12 above)
Comment (kprosise) [reply source/reference-manual/factory/factory-definition.rst:120]:
```suggestion
        If needed, `ask a support engineer <support.foundries.io>`_ to update the machine definition for your Factory.
```

File: source/reference-manual/factory/factory-definition.rst (hunk 13 above)
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:127]:
```suggestion
      from ``recipes-samples/images/lmp-factory-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```

File: source/reference-manual/factory/factory-definition.rst (hunk 14 above)
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:145]:
```suggestion
                  IMAGE: lmp-factory-image
```

File: source/reference-manual/factory/factory-definition.rst [hunk 16]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
  image_type:``<lmp_image_type>``
-      **Optional:** Set the LmP image type to produce by recipe name. For
-      example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.
+      **Optional:** Set the LmP image type by recipe name.
+      For example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.
 
-      **Default:** ``lmp-factory-image`` |br| (from
-      **recipes-samples/images/lmp-factory-image.bb** in your
-      **meta-subscriber-overrides.git** repo)
+      **Default:** ``lmp-base-console-image`` |br|
```
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:126]:
```suggestion
      **Default:** ``lmp-factory-image`` |br| 
```

File: source/reference-manual/factory/factory-definition.rst [hunk 17]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
       **Optional:** Control how OTA_LITE tags are handled. See
       :ref:`ref-advanced-tagging` for more details.
 
  mfg_tools:|br| ``- machine: <machine>``
-      **Optional:** Do an OE build to produce manufacturing tooling for a given
-      ``MACHINE``. This is used to facilitate the manufacturing process and to ensure
-      secure boot on devices. Currently only NXP tools are supported.**
+      **Optional:** Do an OE build to produce manufacturing tooling for a given ``MACHINE``.
+      This is used to facilitate the manufacturing process, and to ensure secure boot on devices.
+      Currently, only NXP tools are supported.
```
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:163]:
```suggestion
      Currently, only NXP® tools are supported.
```

File: source/reference-manual/factory/factory-definition.rst (hunk 15 above)
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:240]:
```suggestion
      **Default:** ``arm,arm64,amd64``. 
```

**Comment by doanac** [comment]
Docs for 40bb1ab6317e65e9785349377c255399831640ee are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2485/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 34ea46b91a7a5ef17a01315fdae9e6774c7f8311 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2486/docs/artifacts/html/index.html

File: source/reference-manual/factory/factory-definition.rst [hunk 18]
```diff
@@ -66,20 +65,20 @@ Variables to be used with the metadata and artifacts.
...
                 EXAMPLE_VARIABLE_1: foo
            tagging:
              refs/heads/main:
                - tag: postmerge
-             refs/heads/devel:
-               - tag: devel
+             refs/heads/feature1:
+               - tag: feature1
            mfg_tools:
              - machine: imx8mmevk
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:83]:
```suggestion
             - machine: imx8mm-lpddr4-evk
```

File: source/reference-manual/factory/factory-definition.rst [hunk 19]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
-      **Optional:** Set the LmP image type to produce by recipe name. For
-      example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.
+      **Optional:** Set the LmP image type by recipe name.
+      For example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.
 
-      **Default:** ``lmp-factory-image`` |br| (from
-      **recipes-samples/images/lmp-factory-image.bb** in your
-      **meta-subscriber-overrides.git** repo)
+      **Default:** ``lmp-factory-image`` |br| 
+      from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:127]:
```suggestion
      from ``recipes-samples/images/lmp-factory-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```

File: source/reference-manual/factory/factory-definition.rst [hunk 20]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
-               - imx8mmevk
-             ref_options:
-               refs/heads/devel:
-                 machines:
-                   - raspberrypi3-64
+	      # In the below example, when the branch named "feature1" is built by our
+	      # CI system, it will have its option values for "machine" and
+	      # "params" overriden by what is specified after "refs/heads/feature1:".
+	      # In the "feature1" build, IMAGE will now equal "lmp-mini-image" rather than
+	      # "lmp-base-console-image" as initially defined.
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-definition.rst:141]:
```suggestion
	      # "lmp-factory-image" as initially defined.
```

File: source/reference-manual/factory/factory-definition.rst [hunk 21]
```diff
@@ -90,86 +89,82 @@ Variables to be used with the metadata and artifacts.
...
+
+               lmp:
                  params:
-                   IMAGE: lmp-mini
+                  IMAGE: lmp-base-console-image
```
Comment (kprosise) [inline source/reference-manual/factory/factory-definition.rst:145]:
```suggestion
                  IMAGE: lmp-factory-image
```

**Comment by doanac** [comment]
Docs for 55be7fdbd499f770b75aa043c218033af6e6f362 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2487/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 5c97cd7385f1955e4c54c61509b49347686f8786 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2488/docs/artifacts/html/index.html

**Pull request merged**